main/entry.c

```diff
@@ -1152,10 +1152,18 @@ extern int makeTagEntry (const tagEntryInfo *const tag)
 	Assert (tag->name != NULL && strchr (tag->name, '\t') == NULL);
 	Assert (getSourceLanguageFileKind() == tag->kind || isSourceLanguageKindEnabled (tag->kind->letter));
 
-	if (tag->name [0] == '\0' && (!getSourceLanguageAllowNullTag()) && (!tag->placeholder))
+	if (tag->name [0] == '\0')
 	{
-		error (WARNING, "ignoring null tag in %s(line: %lu)", vStringValue (File.name), tag->lineNumber);
-		goto out;
+		if (tag->placeholder)
```

Comment:
this could be kept as part of the above `if`: `if (tag->name [0] == '\0' && (!tag->placeholder))`